flatpak-gtk.c

```diff
@@ -458,6 +460,12 @@ handle_file_chooser_open (FlatpakDesktopFileChooser *object,
         }
       g_variant_iter_free (iter);
     }
+  if (g_variant_lookup (arg_options, "current_name", "&s", &current_name))
+    gtk_file_chooser_set_current_name (GTK_FILE_CHOOSER (dialog), current_name);
+  if (g_variant_lookup (arg_options, "current_folder", "&ay", &path))
+    gtk_file_chooser_set_current_folder (GTK_FILE_CHOOSER (dialog), path);
```

Comment:
How would the client know what to send for current_folder though? We generally hide the full path of the selected object from the client when we return a document uri during file selection.